test/unit/TestH26xRTPPacketization.cpp

```diff
@@ -0,0 +1,298 @@
+#include "TestCommon.h"
+#include "h264/H264Packetizer.h"
+#include "h265/H265Packetizer.h"
+
+class H264PacketizerForTest : public H264Packetizer
+{
+    public:
+        void TestRTPFUA(VideoFrame& frame, BufferReader nal)
+        {
+            H264Packetizer::EmitNal(frame, nal);
+        }
+};
+
+class H265PacketizerForTest : public H265Packetizer
+{
+    public:
+        void TestRTPFUA(VideoFrame& frame, BufferReader nal)
+        {
+            H265Packetizer::EmitNal(frame, nal);
+        }
+};
+
+TEST(TestFUA, h264Fragmentation)
+{
+    static constexpr int H264NalHeaderSize = 1;
+    static constexpr int FUPayloadHdrSize = 1;
+    static constexpr int FUHeaderSize = 1;
+    H264PacketizerForTest h264Packetizer;
+
+    {
+        // min nal payload size for FU-A
+        static constexpr int nalSize = RTPPAYLOADSIZE;
+        static constexpr int nalPayloadSize = RTPPAYLOADSIZE - H264NalHeaderSize;
+        uint8_t nalData[nalSize] = {0};
+        nalData[0] = 0x65;
+
+        VideoFrame frame(VideoCodec::H264);
+        BufferReader nal(nalData, nalSize);
+
+        h264Packetizer.TestRTPFUA(frame, nal);
+
+        int expectedPos = 4 + H264NalHeaderSize;
+        int expectedRTPPacekts = std::ceil((double)nalPayloadSize / (RTPPAYLOADSIZE - FUPayloadHdrSize - FUHeaderSize));
+        auto packetLen = nalPayloadSize / expectedRTPPacekts;
+        int mod = nalPayloadSize % expectedRTPPacekts;
+        EXPECT_TRUE(mod!=0);
```

Comment:
Consider use:
```suggestion
        ASSERT_TRUE(0, mod);
```
It would give more useful info if it fails.